manifests/init.pp

```diff
@@ -28,6 +28,7 @@
   $ssh_config_macs                            = undef,
   $ssh_config_use_roaming                     = 'USE_DEFAULTS',
   $ssh_config_template                        = 'ssh/ssh_config.erb',
+  $ssh_config_identityfiles                   = ['~/.ssh/id_rsa', '~/.ssh/id_dsa'],
```

Comment:
You could use a larger set as default value.
The list from the ssh_config man page is much larger:
```
~/.ssh/id_dsa
~/.ssh/id_ecdsa
~/.ssh/id_ecdsa_sk
~/.ssh/id_ed25519
~/.ssh/id_ed25519_sk
~/.ssh/id_rsa
```